leases/schema/mutations.py

```diff
@@ -411,6 +413,47 @@ def mutate_and_get_payload(cls, root, info, **input):
         return SetStickersPostedMutation()
 
 
+class SendExistingInvoicesInput:
+    due_date = graphene.Date(
+        description="Defaults to 14 days from the date when the mutation is executed."
+    )
+
+
+class SendExistingBerthInvoicesMutation(graphene.ClientIDMutation):
+    class Input(SendExistingInvoicesInput):
+        pass
+
+    result = graphene.Field(SendExistingInvoicesType)
+
+    @classmethod
+    @view_permission_required(CustomerProfile)
+    @change_permission_required(BerthLease, WinterStorageLease, Order)
+    def mutate_and_get_payload(cls, root, info, **input):
+        from payments.schema import OrderNode
```

Comment:
Can this be moved to start of the file with other imports?

Reply:
It causes some circular import errors, that's why I added the import here.